nexus/src/external_api/http_entrypoints.rs

```diff
@@ -8924,6 +8992,45 @@ async fn session_sshkey_list(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
+/// List SSH public keys
+///
+/// Lists SSH public keys for the currently authenticated user.
+#[endpoint {
+    method = GET,
+    path = "/v1/current-user/sshkeys",
```

Comment:
ssh-keys? kabob seems to be our preference for multi-word path components?

Reply:
Yep, I think that was just a mistake on my part.